lib/cocoapods/validator.rb

```diff
@@ -610,18 +657,17 @@ def parse_xcodebuild_output(output)
     #         returns its output (both STDOUT and STDERR).
     #
     def xcodebuild
-      command = 'xcodebuild clean build -target Pods'
-
+      command = %W(clean build -workspace App.xcworkspace -scheme App)
       case consumer.platform_name
       when :ios
-        command << ' CODE_SIGN_IDENTITY=- -sdk iphonesimulator'
+        command += %w(CODE_SIGN_IDENTITY=- -sdk iphonesimulator)
       when :watchos
-        command << ' CODE_SIGN_IDENTITY=- -sdk watchsimulator'
+        command += %w(CODE_SIGN_IDENTITY=- -sdk watchsimulator)
       when :tvos
-        command << ' CODE_SIGN_IDENTITY=- -sdk appletvsimulator'
+        command += %w(CODE_SIGN_IDENTITY=- -sdk appletvsimulator)
       end
 
-      output, status = _xcodebuild "#{command} 2>&1"
+      output, status = Dir.chdir(validation_dir) { _xcodebuild(command) }
```

Comment:
we can probably get rid of the `chdir` by passing an absolute path to the workspace?